README.md

```diff
@@ -667,7 +667,46 @@ make run-tests
 
 This will run a set of both unit `(TODO)` and integration tests. See the `Makefile` for more details
 
+<br />
+
+## Local packages development
```

Comment:
retitle: `Local NPM package development` and move to a 3rd-level heading under the `Development` header, without the `<br />`